src/main/cpp/commands/drivetrain/TurnCommand.cpp

```diff
@@ -8,21 +8,31 @@ TurnCommand::TurnCommand(std::shared_ptr<Drivetrain> drivetrain, double angle,
     angle_ = angle;
     max_output_ = max_output;
 
-    this->AddRequirements(drivetrain_.get());
+    if (drivetrain_ != nullptr) {
+        this->AddRequirements(drivetrain_.get());
+    }
```

Comment:
I know this is off-season so it’s fine but should changes like all the drivetrain command ones be checked out to a separate branch or something? Feels like there should be a separation between this shooter PR and bugfixes like that

Reply:
Yeah this is a "drive by change" and in the regular season we should not be doing this lol